mirgecom/profiling.py

```diff
@@ -315,6 +362,14 @@ def _cache_kernel_stats(self, program: lp.kernel.LoopKernel, kwargs: dict) \
                 footprint_bytes=footprint_bytes)
 
             self.kernel_stats.setdefault(program, {})[args_tuple] = res
+
+            if self.logmgr:
+                if "pyopencl_array_time" not in self.logmgr.quantity_data:
+                    self.logmgr.add_quantity(KernelProfile(self, "pyopencl_array"))
```

Comment:
Missed it earlier, I'll nag about it here: These should be identified by their names.

Reply:
This will be fixed automatically with #193.

Reply:
Should be fixed now.